code/modules/mob/holder.dm

```diff
@@ -4,66 +4,282 @@ var/list/holder_mob_icon_cache = list()
 /obj/item/weapon/holder
```

Comment:
This entire file was replaced wholesale. And i went back to test that every holdable mob still worked as before.
I can't recall which ones were pickuppable before but corgis now are at least, that one is new

Tons of new stuff in this file